src/uucore/src/lib/features/perms.rs

```diff
@@ -487,32 +488,58 @@ impl ChownExecutor {
     }
 }
 
+/// common options
```

Comment:
seems useless
can we ignore this ?

Reply:
Yes

But when we will add `#![deny(missing_docs)]` everything will be documented ^^